### PR TITLE
refactor(building-entrance): refactor to ModelForm

### PR DESCRIPTION
**Pull request by velrest**

### CODE REVIEW

File: addon/templates/building/edit/entrance/edit/index.hbs [hunk 1]
```diff
@@ -17,60 +17,59 @@
...
-        <Field @attr="EDID" type="number" readonly disabled />
+        <Field @attr="EDID" @type="number" @disabled={{true}} />
       {{/unless}}
       <Field
         @attr="locality.name.nameLong"
-        required
-        readonly={{not this.buildingEntrance.isNew}}
-        disabled={{not this.buildingEntrance.isNew}}
+        @required={{true}}
+        @readonly={{not this.buildingEntrance.isNew}}
```
Comment (luytena) [inline addon/templates/building/edit/entrance/edit/index.hbs:35]: I don't think there is a `@readonly` parameter (unless you added it in the meantime)

Comment (velrest) [reply addon/templates/building/edit/entrance/edit/index.hbs]: No there isn't :sweat_smile: i just added a `@` to every argument.

File: translations/building-entrance/de.yaml [hunk 2]
```diff
@@ -9,14 +9,15 @@ ember-gwr:
     saveSuccess: Eingang erfolgreich gespeichert.
     loadingError: Beim Laden des Eingangs ist ein Fehler aufgetretten.
     street: Strasse
-    coordinates: Koordinaten
+    streetHint: Um einen Eingang zu erfassen müssen Sie zuerst eine Strasse verknüpfen.
+    coordinates: KoordinateStrassen
```
Comment (luytena) [inline translations/building-entrance/de.yaml:13]: Missing space?

**Review by luytena — APPROVED** [review]
Otherwise looks good to me :)

File: translations/building-entrance/de.yaml (hunk 2 above)
Comment (velrest) [reply translations/building-entrance/de.yaml]: Done

Comment (luytena) [reply translations/building-entrance/de.yaml]: :ok_hand:

**Pull request merged**